### DISCUSSION TITLE
Welcome to AutoGPT-Code-Ability Discussions!

### DISCUSSION THREAD

**Discussion by aarushik93** [Significant-Gravitas/AutoGPT-Code-Ability#291] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Toocky** [comment]
Hi aarushik93, 

Have been following the AutoGPT community for a while, we were more active earlier on and built one of the first UI's  https://github.com/neuronic-ai/autogpt-ui


More recently we've been building our API service out and released our Beta https://apipie.ai/

I am keen to see what this tool becomes, have posted some ideas we've used for a similar tool.
